styleguide/components/41-organisms/teaser-collections/teaser-topic-collection/_teaser-topic-collection.scss

```diff
@@ -0,0 +1,19 @@
+.teaser-topic-collection {
+  ul {
+    @extend %list-no-style;
+    @supports (display: grid) {
```

Comment:
You use @support, but provide no fallback for when there is no support.
Have you checked this in IE?

Reply:
@delrueba It uses flexbox now for better browser support.